### DISCUSSION TITLE
The battery charging limit function of M16 (2021) is invalid.

### DISCUSSION THREAD

**Discussion by ghost** [seerge/g-helper#1176] [Q&A]
The battery charging limit function of M16 (2021) is invalid
Dear seerge:
  Thank you very much for using your software, which has allowed me to relax in my difficulty with the annoying sound of the computer. However, the issue of battery charging restrictions was found during use. Sometimes when the computer is turned on and the software has not automatically started, the battery will charge. Sometimes, the next day after shutting down, it shows 100% battery, which makes me confused. What method do you have for this problem?
  Finally, thank you very much for your development. I will recommend G-Helper to more friends who use ROG series products

**Comment by seerge** [comment]
@SSSMaoYi hello,

Some models start charging on reboot, this is how it's done on hardware level. 
As soon as app starts with windows it will set a limit again.

**Reply by ghost** [reply]
Wow, your response speed shocked me! Following your answer, I would like to ask: Can I achieve the goal of limiting to 60% again by manually discharging? I agree with half of your answer because based on my current observation, my computer M16 (2021) is available before the G-helper version and bios update.

**Reply by seerge** [reply]
Yes, you can manually discharge of course if you will unplug.

**Reply by ghost** [reply]
Haha, thank you for your Q&A! The communication with you is very interesting, and I hope to have new contact with you. Wishing you a wonderful night!
